test/run-webhook.py

```diff
@@ -64,7 +64,7 @@
         "uid": "00000000-0000-0000-0000-000000000000"
     },
     "spec": {
-        "protocol": {
+        "discoveryHandler": {
             "debugEcho": {
```

Comment:
from memory, this doesn't seem to be the new style of configuration

Reply:
nice catch! fixed b0d7fba90f02